### DISCUSSION TITLE
Blocking or retrieving information about forwarding bot's messages

### DISCUSSION THREAD

**Discussion by kapisolec** [telegraf/telegraf#1754] [Q&A] [answered]
Hi, I was thinking, is there a way to block users from forwarding telegram bot messages or retrieving information about forwarded messages of the bot?

**Comment by MKRhere** [comment, marked as answer]
See `protect_content` in send* methods: https://core.telegram.org/bots/api#sendmessage

```TS
ctx.sendMessage("Protected reply", { protected_content: true });
```

**Reply by zhaody99** [reply]
Setting protect_content to true can prevent messages from being forwarded and saved. However, if the owner creates a discuss group which link to this channel, the message will also be able to forward to the discuss group, the then messages can be forwarded from discuss group. How to prevent this happens? Thanks

**Reply by MKRhere** [reply]
Just a hole on Telegram's side. Nothing we can do except unlink and have the bot post in both places if needed.
